website/docs/r/event_orchestration_unrouted.html.markdown

```diff
@@ -0,0 +1,107 @@
+  The first rule matches only if the condition is matched. The second rule matches always as there are no conditions.
+
+  `catch_all` with empty `actions` block suppresses the alerts that do not match any rules. Using the `catch_all` to set severity, event_action, variables and extractions is also allowed.
+
+```hcl
+resource "pagerduty_event_orchestration_unrouted" "unrouted" {
+ event_orchestration = pagerduty_event_orchestration.my_monitor.id
+  sets {
+    id = "start"
+    rules {
+      label = "Update the summary of un-matched Critical alerts so they're easier to spot"
+      conditions {
+        expression = "event.severity matches 'critical'"
+      }
+      actions {
+        severity = "info"
+        extractions {
+            target = "event.summary"
+            template = "[Critical Unrouted] {{event.summary}}"
+        }
+      }
+    }
+    rules {
+      label = "Reduce the severity of all other unrouted events"
+      actions {
+        severity = "info"
+      }
+    }
+  }
+  catch_all {
+    actions { }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `event_orchestration` - (Required) The Event Orchestration to which this Unrouted Orchestration belongs to.
+* `sets` - (Required) An Unrouted Orchestration must contain at least a "start" set, but can contain any number of additional sets that are routed to by other rules to form a directional graph.
+* `catch_all` - (Required) When none of the rules match an event, the event will be routed according to the catch_all settings.
+
+### Sets (`sets`) supports the following:
+* `id` - (Required) The ID of this set of rules. Rules in other sets can route events into this set using the rule's `route_to` property.
+* `rules` - (Optional) The Unrouted Orchestration evaluates Events against these Rules, one at a time, and routes each Event based on the first rule that matches. If no rules are provided as part of Terraform configuration, the API returns empty list of rules.
+
+### Rules (`rules`) supports the following:
+* `label` - (Optional) A description of this rule's purpose.
+* `conditions` - (Optional) Each of these conditions is evaluated to check if an event matches this rule. The rule is considered a match if any of these conditions match. If none are provided, the event will `always` match against the rule.
+* `actions` - (Required) Actions that will be taken to change the resulting alert and incident, when an event matches this rule.
+* `disabled` - (Optional) Indicates whether the rule is disabled and would therefore not be evaluated.
+
+### Conditions (`conditions`) supports the following:
+* `expression`- (Required) A [PCL condition](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview) string.
+
+### Actions (`actions`) supports the following:
+* `route_to` - (Required) The ID of the target Service for the resulting alert.
```

Comment:
**Blocker:** Nope, that's the router wording. Also it's Optional. Maybe this instead, taken from the REST API docs?

```suggestion
* `route_to` - (Optional) The ID of a Set from this Unrouted Orchestration whose rules you also want to use with event that match this rule.
```